spec/support/database.rb

```diff
@@ -10,6 +10,7 @@
     t.datetime :persisted_time
     t.date :persisted_date
     t.boolean :persisted_boolean
+    t.string :type
```

Comment:
I'd prefer if we did not have a `type` column here. For the one test where we need it, we could use the `sti_records` table, or add a new table. I'm a bit afraid we might miss some STI bugs in the future, if all our tables have a `type` column which is not a normal situation.

Reply:
Well, when I added the type column a whole bunch of tests failed, not just the new one I added.

I'm not very familiar with rspec. Is there a way to do the same set of tests with different classes? STI classes should behave just like non-STI classes.

Reply:
I see, that's a valid point.
I would still prefer to have a single test (or a few specific tests) for "ActiveType does not treat a `type` column differently anymore", instead of having all tests run with the uncommon situation of a non-STI `type` column, even if this means slightly less exhaustive tests.

There are ways to make a subset of examples run with different parameters which could be used for running the tests with different classes, but I don't think the additional complication would be worth it.

Reply:
OK. I'll rework this then. It'll take me a couple of days though